web/client-api/src/test/java/io/deephaven/web/client/api/ColumnStatisticsTestGwt.java

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.web.client.api;
+
+import elemental2.core.JsMap;
+import elemental2.promise.Promise;
+
+public class ColumnStatisticsTestGwt extends AbstractAsyncGwtTestCase {
+    private final TableSourceBuilder tables = new TableSourceBuilder()
+            .script("from deephaven import empty_table")
+            .script("staticTable", "empty_table(3).update([\"A = i % 2\", \"B = `` + A\"])");
```

Comment:
You could use single quotes instead of escaping the inner double quotes since this looks like a Python script

Reply:
True, this was pasted verbatim from the web IDE as I experimented with the values, and intellij auto-escaped as needed.